src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/model/OffenderImage.java

```diff
@@ -62,8 +60,7 @@
     private Long sequence;
 
     @Column(name = "ACTIVE_FLAG")
-    @Enumerated(EnumType.STRING)
-    private ActiveFlag activeFlag;
+    private String activeFlag;
```

Comment:
could this just be mapped as a boolean so you don't have to do the `= 'Y'` below?